src/engraving/utests/beam_data/beamNoSlope.mscx

```diff
@@ -24,53 +24,6 @@
     <metaTag name="translator"></metaTag>
     <metaTag name="workNumber"></metaTag>
     <metaTag name="workTitle"></metaTag>
-    <Order id="orchestral">
```

Comment:
Could you please explain how this will work now? I'm curious where the instrument order information will now be stored

Reply:
There were only a few utest files that had this `<Order>` element in them; the vast majority don't. Since these tests aren't meant to test that object, there's no reason to have it in there. `beamNoSlope` for example shouldn't be concerned with the `<Order>` element at all. This doesn't reflect how normal scores are saved, this is just a change to make the utests more consistent.